pootle/apps/pootle_project/models.py

```diff
@@ -595,11 +600,21 @@ def invalidate_accessible_projects_cache(sender, instance, **kwargs):
 
     # FIXME: use Redis directly to clear these caches effectively
 
-    cache.delete_many([
-        make_method_key('Project', 'cached_dict', {'is_admin': False}),
-        make_method_key('Project', 'cached_dict', {'is_admin': True}),
-    ])
-
     User = get_user_model()
-    users_list = User.objects.values_list('username', flat=True)
+    users_list = User.objects.filter(is_superuser=False) \
+                             .values_list('username', flat=True)
+    cache.delete_many(
```

Comment:
Thanks @julen. Cool. I thought that it would use [KEYS](http://redis.io/commands/keys) (which can be inefficient), but it' will use [SCAN](http://redis.io/commands/scan).